lte/gateway/python/magma/pipelined/app/enforcement_stats.py

```diff
@@ -97,6 +106,17 @@ def __init__(self, *args, **kwargs):
         if self._print_grpc_payload is None:
             self._print_grpc_payload = \
                 kwargs['config'].get('magma_print_grpc_payload', False)
+        self.ng_config=self._get_ng_config(kwargs['config'], kwargs['rpc_stubs'])
+
```

Comment:
can you also take care of linter issues, some of them looks legit.

Reply:
Changes done